common/test/acceptance/pages/lms/problem.py

```diff
@@ -136,10 +135,33 @@ def click_show(self):
         self.q(css='.problem .show').click()
         self.wait_for_ajax()
 
+    def is_save_notification_visible(self):
+        """
+        Is the Save Notification Visible?
+        """
+        return self.q(css='.notification.warning.notification-save').visible
+
+    def wait_for_save_notification_visible(self):
```

Comment:
I am going to reduce this to 2 methods, one waiting for the visibility and focus and another to check for the visibility.  I use both separately in the test.